VERSION

```diff
@@ -1 +1 @@
-1.18.0
+1.18.1
```

Comment:
why version has changed ?

Reply:
I updated the version for having introduced a change but i could revert it back and update the branch. I however kindly ask you to guide me on when exactly its appropriate for me to update the version file

Reply:
i have reverted to the previous version

Reply:
we change this version only when make release